### PR TITLE
[DOCFIX] Improve doc for proto build

### PR DESCRIPTION
**Pull request by jiacheliu3**
Improves the expression and formats slightly. Also mention in IDE integration.

### CODE REVIEW

File: docs/en/contributor/Building-Alluxio-From-Source.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ $ mvn -T 2C clean install -DskipTests -Dmaven.javadoc.skip -Dfindbugs.skip \
   -Dcheckstyle.skip -Dlicense.skip -Dskip.protoc
 ```
 
-Note: The flag `-Dskip.protoc` means protoc do not work this time, so it won't generate the
-proto relates source file. So you shouldn't use `-Dskip.protoc` flag at the first build.
+> Note: The flag `-Dskip.protoc` skips generating source files related to gRPC proto.
```
Comment (gpang) [inline docs/en/contributor/Building-Alluxio-From-Source.md]: Do you think we also have to mention something about ides and the generated-sources directory?

Comment (jiacheliu3) [reply docs/en/contributor/Building-Alluxio-From-Source.md]: Yes!

And I think that fits better to `Contributor-Tools.md`. PTAL at the new changes added.

File: docs/en/contributor/contributor-tools.md [hunk 2]
```diff
@@ -31,6 +31,9 @@ You may also have to add the classpath variable `M2_REPO` by running:
 $ mvn -Declipse.workspace="your Eclipse Workspace" eclipse:configure-workspace
 ```
 
+> Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target`.
+You will need to mark the directory as a source folder for Eclipse to resolve the sources files.
```
Comment (jiacheliu3) [inline docs/en/contributor/contributor-tools.md]: This is very general as I haven't used Eclipse for years...

File: docs/en/contributor/contributor-tools.md [hunk 3]
```diff
@@ -46,6 +49,9 @@ You can do this by going to
 
 And then checking the box next to "developer" in the window pane. 
 
+> Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target`.
```
Comment (gpang) [inline docs/en/contributor/contributor-tools.md]: should this be
```suggestion
> Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target/generated-sources/protobuf/`.
```

File: docs/en/contributor/contributor-tools.md [hunk 4]
```diff
@@ -31,6 +31,9 @@ You may also have to add the classpath variable `M2_REPO` by running:
 $ mvn -Declipse.workspace="your Eclipse Workspace" eclipse:configure-workspace
 ```
 
+> Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target`.
```
Comment (gpang) [inline docs/en/contributor/contributor-tools.md]: should this be:
```suggestion
> Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target/generated-sources/protobuf/`.
```

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**